src/main/java/org/zalando/nakadi/config/JsonConfig.java

```diff
@@ -35,7 +36,7 @@
     @Primary
     public ObjectMapper jacksonObjectMapper() {
         final ObjectMapper objectMapper = new ObjectMapper()
-                .setPropertyNamingStrategy(CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+                .setPropertyNamingStrategy(SNAKE_CASE);
```

Comment:
will not it break anything from existing behavior?

Reply:
I modified it because the old constant has been deprecated and replaced by this new one. They mean the same thing.